src/core_ocean/Registry.xml

```diff
@@ -3171,6 +3218,52 @@
 			 description="surface pressure forcing due to weight of frazil ice"
 			 packages="frazilIce"
 		/>
+
+		<!-- Fields for tidal potential forcing -->
+		<var name="tidalPotentialEta" type="real" dimensions="nCells Time" units="m"
+			description="Equilibrium tidal potential"
+			packages="forwardMode"
+		/>
+		<var name="tidalPotentialZMid" type="real" dimensions="nVertLevels nCells Time" units="m"
+			description="zMid - Equilibrium tidal potential"
+			packages="tidalPotentialForcingPKG"
```

Comment:
Thanks for putting all your 3D variables in packages. This is the right way to do it.

Reply:
Sure thing!